taxa/taxonomy/taxonomycleaner.php

```diff
@@ -103,6 +103,7 @@ function toggle(divName){
 		?>
 		<!-- inner text block -->
 		<div id="innertext">
+			<h1 class="page-heading"><?= $LANG['TAX_NAME_CLEANER']; ?></h1>
```

Comment:
I think there's already an h1 here
![image](https://github.com/BioKIC/Symbiota/assets/16688784/a04ce3bf-cce0-46ed-80c8-135150b75af9)

Reply:
Fixed in 49e8ca5cc.